File: cloudwatch-alerting-sdk/java-sdk/siren-core/src/main/java/com/stagwelltech/siren/model/Metric.java
package com.stagwelltech.siren.model;

import com.google.common.collect.Maps;
import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.util.Date;
import java.util.Map;

@Data
@NoArgsConstructor
@AllArgsConstructor
@Builder(builderClassName = "Builder")
public class Metric {

    private String category;
    private String name;
    private Map<String, String> tags = Maps.newHashMap();
    private Double value;
    private Date ts;

}

File: cloudwatch-alerting-sdk/dotnet-sdk/net-standard/src/main/siren/Models/Metric.cs
using System;
using System.Collections.Generic;

namespace StagwellTech.SirenSDK.Models
{
    public class Metric
    {
        public Metric() { }
        public Metric(string name, double value)
        {
            this.name = name;
            this.value = value;
        }

        //public string category;
        public string name;
        //public Dictionary<string, string> tags = new Dictionary<string, string>();
        public double value;
        //public DateTime ts = DateTime.Now;

    }
}
